src/blocks.js

```diff
@@ -1852,16 +1852,23 @@
     "block_icon": "./blocks/yashraj2003e/icon.png",
     "author_name": "Yashraj",
     "author_github_url": "https://github.com/yashraj2003e/"
+  },
```

Comment:
This file should not be manually edited, please refer to the Readme for more information and revert this change.